From e6ca6521b4e880c6a314a880e73039e574a02c28 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 23 May 2012 04:22:32 +0000
Subject: Un-typedef model_lookup_t

Long-term, structs will be structs without typedef. This one is the easiest
one to convert, the rest follows later.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
Reviewed-by: Chase Douglas <chase.douglas@canonical.com>
---
(limited to 'src/eventcomm.c')

diff --git a/src/eventcomm.c b/src/eventcomm.c
index 6cfbcc7..0dd789a 100644
--- a/src/eventcomm.c
+++ b/src/eventcomm.c
@@ -274,15 +274,16 @@ event_query_is_touchpad(int fd, BOOL test_grab)
     return (ret == TRUE);
 }
 
-typedef struct {
+#define PRODUCT_ANY 0x0000
+
+struct model_lookup_t {
     short vendor;
     short product;
     enum TouchpadModel model;
-} model_lookup_t;
+};
 
-#define PRODUCT_ANY 0x0000
 
-static model_lookup_t model_lookup_table[] = {
+static struct model_lookup_t model_lookup_table[] = {
     {0x0002, 0x0007, MODEL_SYNAPTICS},
     {0x0002, 0x0008, MODEL_ALPS},
     {0x05ac, PRODUCT_ANY, MODEL_APPLETOUCH},
@@ -306,7 +307,7 @@ event_query_model(int fd, enum TouchpadModel *model_out,
 {
     struct input_id id;
     int rc;
-    model_lookup_t *model_lookup;
+    struct model_lookup_t *model_lookup;
 
     SYSCALL(rc = ioctl(fd, EVIOCGID, &id));
     if (rc < 0)
--
cgit v0.9.0.2-2-gbebe
